algorithms/facenet/vectorize.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+from typing import List
+
+import numpy as np
+import tensorflow as tf
+from facenet_sandberg import face
+
+
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+tf.logging.set_verbosity(tf.logging.ERROR)
+
+
+def get_face_vectors(img_path: str) -> List[List[float]]:
+    identifier = identifier = face.Identifier(
+        facenet_model_checkpoint='20180402-114759.pb')
+    image: np.ndarray = identifier.get_image_from_path(img_path)
+    vectors: List[np.ndarray] = identifier.vectorize(image)
```

Comment:
Please make sure to handle the new `PREALIGNED` flag so that the container can process images that contain just a cropped face (if the flag is set to true) as well as a raw image (if the flag is set to anything else).

Reply:
fixed